exercises/02-robot-syntax.md

```diff
@@ -88,18 +88,21 @@ Login                                                                 | FAIL |
 The reason Robot thinks we have 6 test cases is because each unindented line in the `Test Cases` table is
 considered as a new test case. When a line is indented by 4 spaces, it indicates to Robot that this
 line belongs to the previous unindented line, be it test case, task, or keyword depending on the table it
-belongs. Each indented line is called a **keyword**.
+belongs to. Each indented line is called a **keyword**.
 
-In order to make Robot understand that want to have 1 test case with 6 steps we need to define it like this:
+In order to make Robot understand that you want to have 1 test case with 6 steps, we need to define it like this:
 
 ```robot
 *** Test Cases ***
 
-Welcome Page Should Be Visible After Successful Login
-    Open browser
-    ...
+Welcome Page Should Be Visible After Successful Login # <-Test Case
+    Open browser # <-Step
+    Navigate to localhost:7272 # <-Step
+    # ... and so on
 ```
 
+## Exercise
```

Comment:
I would have only one `## Exercise` in the exercise file to keep it consistent with [all other academy course](https://github.com/orgs/eficode-academy/repositories). Additional sub headers can be added to structure the exercise better, but the header shouldn't be repeated.